bin/load-vault-env

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+
+# Load environment variables from all files in /vault/secrets, if any exist
+if compgen -G "/vault/secrets/*" > /dev/null; then
```

Comment:
Wondering if this makes as much sense as doing it in code (if that would be possible)
Feels like there are other places where this will be needed (plugin-server, celery) and I'm not sure if this is covered by this change alone?

Reply:
I wanted to start with the `posthog-web` deployment to test it and have it running on dev.

But we will add the same on the rest of the entry point scripts.

If we do this on the code then we will be locked by vault.